tests/v3/test_properties.py

```diff
@@ -32,6 +37,11 @@ def test_basic_indexing(data):
 
 
 @given(data=st.data())
+# The filter warning here is to silence an occasional warning in NDBuffer.all_equal
+# See https://github.com/zarr-developers/zarr-python/pull/2118#issuecomment-2310280899
+# Uncomment the next line to reproduce the original failure.
+# @reproduce_failure('6.111.2', b'AXicY2FgZGRAB/8/eLmF7qr/C5EDADZUBRM=')
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
```

Comment:
@dcherian - would you mind looking into this?

Reply:
It's a minefield 😬 The deepest I got was `hypothesis.extra.numpy:ArrayStrategy.set_element`. Something about the sequence of operations and the data passed into the ndarray made it "weird", such that `array == np.complex128(0.0)` raised an InvalidComparision warning. I'm not sure how to interpret the bytes at https://github.com/zarr-developers/zarr-python/pull/2118#issuecomment-2310280899.